apps/ima/translations/src/en/validation.json

```diff
@@ -1,5 +1,59 @@
 {
   "who-are-you": {
     "required": "You must select an option"
+  },
+  "entered-without-permission":{
+    "required": "You must select an option"
+  },
+  "entered-without-permission-detail":{
+    "required": "You must provide an explanation",
+    "maxlength": "Explanation must be 15,000 characters or less",
+    "regex": "Explanation must not include the following special characters [ ] < > / |",
```

Comment:
Did you want to remove this?

Reply:
I've now removed the regex property for all fields in the validation.json file.